extensions/hadoop-dist/files-azure/pom.xml

```diff
@@ -53,4 +53,29 @@
             <version>${hadoop.version}</version>
         </dependency>
     </dependencies>
+
+    <!-- The following part of the POM is related to handling CVE-2019-10172 in jackson-mapper-asl version
+        1.9.13 (dependency of hadoop-azure 3.3.1) -->
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.codehaus.jackson</groupId>
+                <artifactId>jackson-mapper-asl</artifactId>
+                <version>1.9.13.redhat-00007</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

Comment:
I wonder if we should move this to regular dependencies so users depending on non-fat jars get this, instead of the vulnerable version.

Reply:
Good idea. I already talked with Ondra about moving the Red Hat repo to hazelcast-root due to other fixed libraries.